spec/lib/saml_idp/controller_spec.rb

```diff
@@ -72,12 +72,24 @@ def head(status, options = {})
     it "should create a SAML Response" do
       saml_response = encode_response(principal)
       response = OneLogin::RubySaml::Response.new(saml_response)
+      name_id_format_email = Saml::XML::Namespaces::Formats::NameId::EMAIL_ADDRESS
```

Comment:
Why is the default EMAIL_ADDRESS, when the story says it should be PERSISTENT?

Reply:
This is an existing test where name_id_format is passed as email. I added this to be more explicit.